stages/org.osbuild.cloud-init.meta.json

```diff
@@ -46,7 +46,12 @@
             "description": "'system_info' configuration section.",
             "minProperties": 1,
             "properties": {
-              "default_user": {
+            "ssh_pwauth": {
+                "type": "boolean",
+                "description": "enable/disable ssh password authentication",
+                "default": false
```

Comment:
Judging from the upstream docs (https://cloudinit.readthedocs.io/en/latest/reference/modules.html#set-passwords), there's no default. The default sshd behavior will depend on the default distribution configuration. Not specifying the option will keep the distro's default unchanged. Specifying it explicitly will enable or disable the option in sshd.

Please remove the `default` property.